modules/core/src/lib/widget-manager.ts

```diff
@@ -0,0 +1,216 @@
+import type Deck from './deck';
+import type Viewport from '../viewports/viewport';
+import type {PickingInfo} from './picking/pick-info';
+import type {MjolnirPointerEvent, MjolnirGestureEvent} from 'mjolnir.js';
+import type Layer from './layer';
+
+import {EVENTS} from './constants';
+
+export interface Widget<PropsT = any> {
+  // / Populated by core when mounted
+  _element?: HTMLDivElement | null;
+  _viewId?: string | null;
+
+  // / Lifecycle hooks
```

Comment:
Why `// /` in the comments? Some prettier side-effect? Just drop the third slash?